lib/common/bitstream.h

```diff
@@ -220,11 +223,14 @@ MEM_STATIC size_t BIT_initCStream(BIT_CStream_t* bitC,
 }
 
 /*! BIT_addBits() :
- *  can add up to 26 bits into `bitC`.
+ *  can add up to 31 bits into `bitC`.
  *  Note : does not check for register overflow ! */
 MEM_STATIC void BIT_addBits(BIT_CStream_t* bitC,
                             size_t value, unsigned nbBits)
 {
+    MEM_STATIC_ASSERT(BIT_MAX_NBBITS == 32);
+    assert(nbBits <= BIT_MAX_NBBITS);
```

Comment:
should it be `<` in this `assert()` ?
It's supposed to add up to 31 bits, but `BIT_MAX_NBBITS == 32`